tests/engines/registries/acceptance/overview/overview-test.ts

```diff
@@ -132,6 +133,37 @@ module('Registries | Acceptance | overview.overview', hooks => {
         tags.forEach(tag => assert.dom(`[data-test-tags-widget-tag="${tag}"]`).exists());
     });
 
+    test('bookmarks work', async assert => {
+        const reg = server.create('registration', {
+            registrationSchema: server.schema.registrationSchemas.find('prereg_challenge'),
+            currentUserPermissions: Object.values(Permission),
+        });
+
+        server.create('collection', { title: 'Bookmarks', bookmarks: true });
+
+        await visit(`/${reg.id}/`);
+
+        assert.dom('[data-test-social-sharing-button]').isVisible();
+        assert.dom('[data-test-bookmarks-button]').isNotVisible();
+
+        await click('[data-test-social-sharing-button]');
+        assert.dom('[data-test-bookmarks-button]').isVisible();
+
+        // Bookmark registration
+        await click('[data-test-bookmarks-button]');
+        await click('[data-test-social-sharing-button]');
+        assert.dom('[data-test-bookmarks-button]').hasText(
```

Comment:
Could you also assert that the mirage collection has (or doesn't have) the linked registration?